packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -108,9 +107,9 @@ const prepareLocalChainFacadeKit = (
             this.facets.makeAccountWatcher,
           );
         },
-        query() {
-          // TODO https://github.com/Agoric/agoric-sdk/pull/9935
-          return asVow(() => Fail`not yet implemented`);
+        /** @type {HostOf<QueryManyFn>} */
```

Comment:
shouldn't this be defined in terms of `Chain['query']`?

(same goes for the other applicable public methods)